.sass-lint.yml

```diff
@@ -35,7 +35,7 @@ rules:
   no-empty-rulesets: 2
   no-extends: 0
   no-ids: 2
-  no-important: 2
+  no-important: 0
```

Comment:
Why do we need to change this linting rule?

Reply:
Because we need to use !important to override an !important coming from the base styling and this rule would not let us.

Reply:
the print file has to override none print styles, but Id prob have a scss lint ignore comment in the print.scss with an explanation maybe

Reply:
It's a broad problem with using nhsuk-frontend, they are using !important so we have to to override:
https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/core/utilities/_typography.scss

`// Utility classes are allowed to use !important;
// so we disable the sass-lint for that rule
// sass-lint:disable no-important`

Reply:
Ah, OK. That makes sense now. Thanks for the reference.

Might be worth adding a comment just stating that the no-important rule is disabled due to the NHS library though.